app/src/main/java/org/mifos/mobile/ui/about/AboutUsScreen.kt

```diff
@@ -0,0 +1,125 @@
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.CopyrightItemContent
+import org.mifos.mobile.core.ui.component.MifosItemCard
+import org.mifos.mobile.core.ui.component.MifosItemContent
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.utils.Constants.LICENSE_LINK
+import org.mifos.mobile.utils.Constants.SOURCE_CODE_LINK
+import org.mifos.mobile.utils.Constants.WEBSITE_LINK
+import org.mifos.mobile.utils.StartActivity
+import java.util.*
+
+@SuppressWarnings("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun AboutUsScreen(viewModel: AboutUsViewModel = viewModel()) {
+    val context = StartActivity(LocalContext.current)
+
+    MifosMobileTheme {
+        Scaffold {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(16.dp, 56.dp, 16.dp, 16.dp)
+            ) {
+                Image(
+                    painter = painterResource(id = R.mipmap.mifos_icon),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .size(100.dp)
+                        .fillMaxWidth()
+                        .align(Alignment.CenterHorizontally)
+                )
+
+                Text(
+                    text = stringResource(id = R.string.app_name),
+                    style = MaterialTheme.typography.headlineMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 8.dp)
+                )
+
+                Text(
+                    text = stringResource(id = R.string.about_app_description),
+                    style = MaterialTheme.typography.titleSmall.copy(),
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 16.dp)
+                )
+
+                LazyColumn {
+                    items(viewModel.aboutUsList) { item ->
+                        MifosItemCard(onClick = {
+                            when (item.name) {
+                                R.string.official_website -> {
```

Comment:
As we have discussed, it is not a good idea to check with strings inside the composable.
Instead, you create an enum class as follows:

```
  enum class AboutUsItemListItemId{
        OfficialWebsite,
        PrivacyPolicy,
        ...
    }
```

Then use this this as a param in `AboutUsItem` data class:

```
data class AboutUsItem(
    val name: Int,
    val iconUrl: Int? = null,
    val itemId: AboutUsItemListItemId
)
```

Then do the checks based on itemId

Reply:
done